orangecontrib/imageanalytics/import_images.py

```diff
@@ -24,6 +24,7 @@
 
 log = logging.getLogger(__name__)
 
+ImgAttrName = "image"
```

Comment:
I suggest not using CamelCase since it is not according to PEP 8. We can treat this one as a constant and have it uppercase

```suggestion
IMG_ATTR_NAME = "image"
```